### PR TITLE
added APSE to VALID_REGIONS

### PR DESCRIPTION
**Pull request by DjoykeAbyah**
Added APSE to valid regions since we have a live endpoint added to TERMINAL_API_ENDPOINTS_MAPPING

### CODE REVIEW

**Review by Kwok-he-Chu — CHANGES_REQUESTED** [review]
Let's add the tests that allow us to spot these changes from now on if we decide to modify it:
- Test for the VALID_REGIONS
- Test for each TERMINAL_API_ENDPOINTS_MAPPING (and whether they are mapping correctly

**Comment by DjoykeAbyah** [comment]
@Kwok-he-Chu,  I've included the unit test. Since India is also part of valid regions I've included the url in Client.php and added it to the mapping.

**Review by ayodejidev — DISMISSED** [review]
Looks good!

File: tests/Unit/RegionTest.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        );
+    }
+
+    public function testTerminalApiEndpointsMapping()
+    {
+        $expected = [
+            "eu" => "https://terminal-api-live.adyen.com",
+            "us" => "https://terminal-api-live-us.adyen.com",
+            "au" => "https://terminal-api-live-au.adyen.com",
+            "in" => "https://terminal-api-live-in.adyen.com",
```
Comment (Kwok-he-Chu) [inline tests/Unit/RegionTest.php:33]: Could you verify whether the`live-in` endpoint is a valid for terminals

Comment (Kwok-he-Chu) [reply tests/Unit/RegionTest.php:33]: It's not a valid one, let's remove this one and update the tests https://docs.adyen.com/point-of-sale/design-your-integration/terminal-api/ @DjoykeAbyah

File: tests/Unit/RegionTest.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        ];
+
+        $this->assertEquals(
+            $expected,
+            Region::TERMINAL_API_ENDPOINTS_MAPPING,
+            "TERMINAL_API_ENDPOINTS_MAPPING should match the expected mappings."
+        );
+    }
+
+    public function testTerminalApiEndpointsExistsForAllRegions()
```
Comment (Kwok-he-Chu) [inline tests/Unit/RegionTest.php:44]: We can remove this test as not every region (as of currently*) has an endpoint yet (e.g. india) and this test will fail -> see https://docs.adyen.com/point-of-sale/design-your-integration/terminal-api/

At some point, we will  (once we're live)

File: tests/Unit/RegionTest.php (hunk 1 above)
Comment (DjoykeAbyah) [reply tests/Unit/RegionTest.php:33]: Done!

**Pull request merged**